lazyflow/operators/generic.py

```diff
@@ -535,19 +535,13 @@ def execute(self, slot, subindex, roi, result):
         key = roi.toSlice()
         data = self.inputs["Input"][key[:-1] + (slice(None),)].wait()
```

Comment:
```suggestion
        *key, c = roi.toSlice()
        data = self.Input[(*key, slice(None))].wait()
```

... and use `c` instead of `key[-1]`.